playbooks/development.yml

```diff
@@ -1,4 +1,12 @@
 ---
+# Note that by relying on this playbook you will end up with a development
+# environment that uses some of roles we use for live instances. You won't have
+# Nginx nor unicorn so will have to manually start the rails running `bundle
+# exec rails server`.
+#
+# Also, there are a number of things you will need to do manually such as
+# installing bundler and creating the development database with `bundle exec
```

Comment:
P.S. If you want to provision a dev setup without requiring _any_ manual steps from the user, take a look here: https://github.com/openfoodfoundation/ofn-devbox/blob/master/setup/provision.yml